src/services/Functional/User.js

```diff
@@ -2,33 +2,33 @@ import UserAPIService from "@/services/API/User.js";
 
 export default {
   reAuthenticate(store) {
-    // return the stored reAuthCall if re-authentication is in process
+    // return the stored promise from the re-authentication call if re-authentication is in process
     if (store.state.auth.isReAuthenticating)
-      return store.state.auth.reAuthenticationCall;
+      return store.state.auth.reAuthenticationPromise;
 
-    // re authentication is in process
+    // set re-authentication as being in process
     store.dispatch("auth/setReAuthenticationState", true);
 
     // start the refreshAccessToken call and save that promise in the variable
```

Comment:
the line after "and save" should be below as mentioned before as well - I think it got missed.